include/swift/Driver/DependencyGraph.h

```diff
@@ -123,8 +123,9 @@ class DependencyGraphImpl {
   /// The set of marked nodes.
   llvm::SmallPtrSet<const void *, 16> Marked;
 
-  /// A list of all "external" dependencies that cannot be resolved just from
-  /// this dependency graph.
+  /// A list of all external dependencies that cannot be resolved
+  /// just from this dependency graph. They must be resolved from definitions
+  /// in files that do not belong to this module.
```

Comment:
This comment is still off in the same way as the ReST file was: the contents of this set are not references to definitions (/ declarations / whatever), but just the files themselves.